app/admin/restaurant.rb

```diff
@@ -1,5 +1,6 @@
 ActiveAdmin.register Restaurant do
 
-  permit_params :name, :address, :description, :restaurant_category_id
+  permit_params :name, :address, :description, :restaurant_category_id, :admin_user_id
```

Comment:
` :admin_user_id` can be ` :admin_user` ??

Reply:
Haven't tried, will do that.